app/controllers/alchemy/pages_controller.rb

```diff
@@ -104,7 +104,7 @@ def load_index_page
     # @return NilClass
     #
     def load_page
-      @page ||= Page.contentpages.find_by(
+      @page ||= Language.current.pages.find_by(
         urlname: params[:urlname],
         language_code: params[:locale] || Language.current.code
```

Comment:
Nope, I thought that too, but its important that it stays.
Language.current is always an actual language, while params[:locale] can be a not existing one. Without that part we could find a page even if the request was for another language.